File: backend/src/main/java/com/netcompany/coe/login/filters/AccessDeniedExceptionMapper.java
package com.netcompany.coe.login.filters;

import com.netcompany.coe.login.exceptions.AccessDeniedException;

import javax.ws.rs.core.Response;
import javax.ws.rs.ext.ExceptionMapper;
import javax.ws.rs.ext.Provider;

@Provider
public class AccessDeniedExceptionMapper implements ExceptionMapper<AccessDeniedException> {
    @Override
    public Response toResponse(final AccessDeniedException e) {
        return Response.status(403).build();
    }
}

File: backend/src/main/java/com/netcompany/coe/login/filters/AccessDeniedExceptionMapper.kt
package com.netcompany.coe.login.filters

import com.netcompany.coe.login.exceptions.AccessDeniedException

import javax.ws.rs.core.Response
import javax.ws.rs.ext.ExceptionMapper
import javax.ws.rs.ext.Provider

@Provider
class AccessDeniedExceptionMapper : ExceptionMapper<AccessDeniedException> {

    override fun toResponse(e: AccessDeniedException): Response {
        return Response.status(403).build()
    }
}
